wpilibc/src/main/native/include/frc/SpeedControllerGroup.h

```diff
@@ -7,26 +7,25 @@
 #include <functional>
 #include <vector>
 
-#include <wpi/deprecated.h>
 #include <wpi/sendable/Sendable.h>
 #include <wpi/sendable/SendableHelper.h>
 
 #include "frc/motorcontrol/MotorController.h"
 
 namespace frc {
 
-class WPI_DEPRECATED("use MotorControllerGroup") SpeedControllerGroup
+class [[deprecated("use MotorControllerGroup")]] SpeedControllerGroup
     : public wpi::Sendable,
       public MotorController,
       public wpi::SendableHelper<SpeedControllerGroup> {
  public:
   template <class... SpeedControllers>
-  explicit SpeedControllerGroup(SpeedController& speedController,
-                                SpeedControllers&... speedControllers);
+  explicit SpeedControllerGroup(SpeedController & speedController,
+                                SpeedControllers & ... speedControllers);
```

Comment:
The annotation seems to be tripping a bug in clang-format, as these should not have changed.